deploy-board/deploy_board/webapp/helpers/placements_helper.py

```diff
@@ -36,3 +36,13 @@ def get_by_provider_and_cell_name(request, provider, cell_name):
 
 def get_by_id(request, placement_id):
     return rodimus_client.get("/placements/%s" % placement_id, request.teletraan_user_id.token)
+
+
+def get_simplified_by_ids(request, placement_str, provider, cell):
+    current_placement_ids = set(placement_str.split(','))
+    all_placements = get_by_provider_and_cell_name(request, provider, cell)
+    current_placements = filter(lambda p: p['id'] in current_placement_ids, all_placements)
+    return map(lambda p: {'id': p['id'],
```

Comment:
I think this is fine, but you might be better served by a list comprehension here that pares extra keys from the original list of dicts, here is a similar example without list comprehension: https://stackoverflow.com/a/12387627

Reply:
you would basically provide the desired keys as a list and make the check `if k not in [a, b, c, d]: del d[k]`